auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -355,7 +347,7 @@ def get_crop_transforms(self):
 
             num_steps_per_image = self.crop_params.get("num_steps_per_image", 1)
             if num_steps_per_image > 1:
-                print('Cropping with num_steps_per_image', num_steps_per_image)
+                print(f"Cropping with num_steps_per_image {num_steps_per_image}")
```

Comment:
What's the `num_steps_per_image` for?

Reply:
it's number of crops to do from the same image (used by RandCrop... transforms)